explanation/web-dashboard.md

```diff
@@ -10,8 +10,7 @@ The dashboard comes pre-installed when you deploy the full version of Anbox Clou
 The dashboard allows multiple views and management of various objects such as applications, instances and nodes to some extent. The operations available from the dashboard may be limited as compared to the CLI, however if you are new to the CLI, the dashboard can be a friendlier start to Anbox Cloud.
 
 * Applications view - Allows creation, streaming and management of applications.
-* Sessions view - Allows a list view of the sessions and their status.
-* Instances view - Allows a detailed view of instances and the applications or images that they are created from, along with management options for the instances.
+* Instances view - Allows a detailed view of instances and the applications or images that they are created from, along with management and streaming options for the instances.
```

Comment:
Would an `other` instead of just `streaming` suffice ? since we will also have the adb connect option there now.

Reply:
Thanks, I still think it is important to give an idea about the kind of options but have made it a bit more non-specific.